Yoast/yoast-components
# Add overall score to the redux state

## Summary

This PR can be summarized in the following changelog entry:

* Adds the overall score for each type of analysis to the redux state

## Relevant technical choices:

* Adds actions and reducers for overall scores to the content analysis reducer

## Test instructions

This PR can be tested by following these steps:

* Run `yarn install` and `yarn test`, and check whether the tests pass
* You can also expose the store and the action to the window in edit.js, and dispatch it (store.dispatch( action() ) and check whether the state was changed correctly regarding the overall scores. Readability results and SEO results should have separate overall scores, and SEO should also have an overall score for each keyword.

Fixes https://github.com/Yoast/wordpress-seo/issues/9308
